docs/users/overview.adoc

```diff
@@ -103,3 +103,31 @@ After navigating around, you may want to reset the zoom level and go back to a f
 image::images/bpmn-diagram_navigation_C.2.0.gif[BPMN Diagram Navigation]
 
 _Navigation with the https://github.com/bpmn-miwg/bpmn-miwg-test-suite/blob/cc75e467fd2b3009e67d4b24943591c66ce91a23/Reference/C.2.0.bpmn[migw-test-suite C.2.0] BPMN diagram_
+
+=== Purpose
+Process Analytics gives you the opportunity to visualize the Process Data on top of the BPMN diagram.
+The purpose of the library is to gain the visibility on what happens in your processes.
```

Comment:
Translation tools propose 'The purpose of the library is to gain visibility into what is going on in your processes' but I am not sure of this sentence

Reply:
hehehe, we must ask Mickey to check it for us:)

for me both make sense.

Reply:
Ok fine, let's keep it as it is now, we will request a review later when all doc update we are working on will be completed.write the whole documentation.